src/configure/configure.ts

```diff
@@ -314,9 +314,9 @@ class PipelineConfigurer {
         try {
             let azureResource: GenericResource = await this.appServiceClient.getAppServiceResource((<AzureTreeItem>node).fullId);
 
-            switch (azureResource.type.toLowerCase()) {
+            switch (azureResource.type ? azureResource.type.toLowerCase() : '') {
```

Comment:
In what cases will the type be null?

Reply:
it is an optional property in the underlying azure resource object we use. It can be null if the response doesn't contain it.
Primarily did this to safeguard for null references with optional properties